### PR TITLE
Jv fix outgoing links

### PR DESCRIPTION
**Pull request by jakubvul-okta**
## Description:
- **What's changed?** Fixed outgoing links that were either 404 or to HOC and not in the proper alias format
- **Is this PR related to a Monolith release?** No
### Resolves:

* N/A

### CODE REVIEW

File: packages/@okta/vuepress-site/code/javascript/okta_auth_sdk/index.md [hunk 1]
```diff
@@ -23,8 +23,8 @@ If you'd like to explore the entire Auth SDK, please see the [Okta AuthJS Source
 You will need the following things for this guide:
 
 - An Okta org - If you don't have an existing org, register for [Okta Developer Edition](https://developer.okta.com/signup/).
-- An OpenID Connect Application. Instructions for creating one can be found on [this page](https://help.okta.com/en/prod/okta_help_CSH.htm#ext_Apps_App_Integration_Wizard), under the "OpenID Connect Wizard" section.
-- At least one User [assigned to the Application](https://help.okta.com/en/prod/Content/Topics/Apps/Apps_Apps_Page.htm#Assigning).
+- An OpenID Connect Application. Instructions for creating one can be found on [this page](https://help.okta.com/en/prod/okta_help_CSH.htm#csh-attribute-statements-saml), under the "OpenID Connect Wizard" section.
```
Comment (IanHakes-okta) [inline packages/@okta/vuepress-site/code/javascript/okta_auth_sdk/index.md:26]: This is not the correct place to link to any more - the instructions for creating an OIDC app with the wizard are at:
https://help.okta.com/en/prod/Content/Topics/Apps/Apps_App_Integration_Wizard_OIDC.htm

Text should be changed to:
`- An OpenID Connect application. Instructions for creating one can be found on [this page](https://help.okta.com/en/prod/okta_help_CSH.htm#ext_Apps_App_Integration_Wizard-oidc)`

File: packages/@okta/vuepress-site/code/javascript/okta_auth_sdk/index.md [hunk 2]
```diff
@@ -23,8 +23,8 @@ If you'd like to explore the entire Auth SDK, please see the [Okta AuthJS Source
...
+- At least one User [assigned to the Application](https://help.okta.com/en/prod/okta_help_CSH.htm#ext-apps-page-show-application-embed-links).
```
Comment (IanHakes-okta) [inline packages/@okta/vuepress-site/code/javascript/okta_auth_sdk/index.md:27]: This is going to be changed shortly when I get time to refactor this page, but I'll take that as a todo to fix the link.

The text should be updated though to 
`- At least one user [assigned to the application]` (no need for caps as these aren't resource objects)

**Review by bretterer — APPROVED** [review]

File: packages/@okta/vuepress-site/docs/release-notes/2018-36/index.md [hunk 3]
```diff
@@ -16,7 +16,7 @@ title: Okta API Products Release Notes
 
 ### New Device Notification Emails are Generally Available (GA)
 
-When enabled, end users will receive a new device notification email when signing in to Okta from a new or unrecognized device. This feature is now generally available to all orgs. For more information about email notifications, refer to the New or Unknown Device Notification Emails section on [this page](https://help.okta.com/en/prod/Content/Topics/Security/Security_General.htm). <!--OKTA-186366-->
+When enabled, end users will receive a new device notification email when signing in to Okta from a new or unrecognized device. This feature is now generally available to all orgs. For more information about email notifications, refer to the New or Unknown Device Notification Emails section on [this page]https://help.okta.com/en/prod/okta_help_CSH.htm#ext_Security_General. <!--OKTA-186366-->
```
Comment (IanHakes-okta) [inline packages/@okta/vuepress-site/docs/release-notes/2018-36/index.md:19]: Parens removed. Should be:
`Unknown Device Notification Emails section on [this page](https://help.okta.com/en/prod/okta_help_CSH.htm#ext_Security_General). <!--OKTA-186366-->`

**Review by IanHakes-okta — APPROVED** [review]
Just 4 changes. I have not downloaded the branch to test all the links, however. Assuming you have and they all work, lgtm

File: packages/@okta/vuepress-site/docs/release-notes/2018-36/index.md (hunk 3 above)
Comment (jakubvul-okta) [inline packages/@okta/vuepress-site/docs/release-notes/2018-36/index.md:19]:
```suggestion
When enabled, end users will receive a new device notification email when signing in to Okta from a new or unrecognized device. This feature is now generally available to all orgs. For more information about email notifications, refer to the New or Unknown Device Notification Emails section on [this page](https://help.okta.com/en/prod/okta_help_CSH.htm#ext_Security_General). <!--OKTA-186366-->
```

File: packages/@okta/vuepress-site/code/javascript/okta_auth_sdk/index.md (hunk 1 above)
Comment (jakubvul-okta) [inline packages/@okta/vuepress-site/code/javascript/okta_auth_sdk/index.md:26]:
```suggestion
- An OpenID Connect Application. Instructions for creating one can be found on [this page](https://help.okta.com/en/prod/okta_help_CSH.htm#ext_Apps_App_Integration_Wizard-oidc), under the "OpenID Connect Wizard" section.
```

File: packages/@okta/vuepress-site/code/javascript/okta_auth_sdk/index.md (hunk 2 above)
Comment (jakubvul-okta) [inline packages/@okta/vuepress-site/code/javascript/okta_auth_sdk/index.md:27]:
```suggestion
- At least one User [assigned to the application](https://help.okta.com/en/prod/okta_help_CSH.htm#ext-apps-page-show-application-embed-links).
```

**Pull request merged**